src/nodeScribbletune.js

```diff
@@ -3,11 +3,11 @@ const scribble = require('scribbletune');
 
 maxApi.addHandler("makeClip", () => { //we create an event handler function to contain the script. For more information on how this works, see Node for Max documantation https://docs.cycling74.com/nodeformax/api/
     
-    const constructChords = (async () => {
+    const constructClip = (async () => {
         const parameters = await maxApi.getDict("parameters") //we fetch the main dictionary "parameters"
         
-        const notes = parameters.notes //we retrieve the parameters
-        const pattern = parameters.pattern
+        const { notes, pattern } = parameters //we retrieve the parameters
```

Comment:
Are only these 2 params used here by default as they are "required" in Scribbletune and the rest are optional? However my 2 cents are that we should bring in all the params of Scribbletune in here by default and let users remove it if they dont want it. This way they can experiment adding newer features to their device using Scribbletune::clip's API without having to worry if a particular param is being imported or not.

I m yet to clone your branch and try it out hence need to quickly ask you if my understanding is correct on the following:
1. src/nodeScribbletune.js will be downloaded to the user's local machine by the "generate" script 
2. users can manipulate it further if they want 
3. users must run some script to produce/update the amxd file

Reply:
To your point on bringing all the params in, do you mean to enrich the UI of the template device with stuff like subdiv, sizzle etc? I think thats probably a good idea.

Reply:
About the installation:

1. thats the idea
2. yes
3. Im affraid this is the biggest potential downside of this whole thing

To make the device ready for distribution, users have to jump through a few hoops, there is a detailed guide [here](https://cycling74.com/forums/node-for-max-bundling-in-collectives-standalones-and-max-for-live-devices). Heres the bad news: only users with full Max licence can do this. That means people who have Max for Live, that is Max licence obtained via purchasing Live Suite licence cannot export so called [collectives](https://docs.cycling74.com/max8/vignettes/collective_guide). Freezing the device works with js obcets, but Im 90% sure it doesent work with npm packages.

So users who own only the M4L licence can either
1. use webpack or some other bundler and Im not sure that will work 100%
2. include the `npm script install` button for first time users and make the end user install all the npm stuff on their machine

In summary, those who download it and want to use it only for their personal use dont need to worry about step 3. Those who want to give or sell devices to other producers need to have full Max license or use Webpack.

Reply:
`To make the device ready for distribution, users have to jump through a few hoops` <- this is perfectly fine Since they are already using npm to install, we can assume they are running node locally and we can provide another script that exports the module for distribution.